spam/01_spam_tutorial.py

```diff
@@ -895,8 +964,7 @@ def plot_probabilities_histogram(Y):
     verbose=0,
 )
 
-Y_probs_test = keras_dev_model.predict(x=X_test)
-Y_preds_test = np.round(Y_probs_test)
+Y_preds_test = np.round(keras_dev_model.predict(x=X_test))
```

Comment:
nit: to avoid people doing weird things when running cells out of order, maybe call this something different than the preds from the label model training labels